### PR TITLE
0 intro bogdandrummer

### PR DESCRIPTION
**Pull request by Bogdandrummer**
First hometask. Just creating my brunch and readme.md file

### CODE REVIEW

**Comment by vladyslav-yermolin-moc** [comment]
Please submit your work in Classroom https://classroom.google.com/c/NjI5NTUxMjg5OTY5/a/NjMwMTMwNjQ0NjQx/details

**Comment by vladyslav-yermolin-moc** [comment]
Please make a branch in our repo and do pull request from it

**Comment by vladyslav-yermolin-moc** [comment]
Please add to the pull request only your folder and Readme file. Now you like to change package-lock.json of whole our project.

File: package-lock.json
Comment (vadym-zinchenko-moc) [inline package-lock.json:1]: Hi @Bogdandrummer, you have to add only README file. Don't touch `package-lock.json` file.

**Review by vadym-zinchenko-moc — CHANGES_REQUESTED** [review]

**Pull request closed without merging**